clients/unshaded/src/test/java/tachyon/client/file/FileSystemContextTest.java

```diff
@@ -48,6 +48,15 @@ public void concurrencyTest() throws Exception {
     for (FileSystemMasterClient client : clients) {
       FileSystemContext.INSTANCE.releaseMasterClient(client);
     }
+
+    // wait for the spawned thread to complete. If it is unable to acquire a master client before
+    // the defined timeout, fail.
+    final long timeoutMs = 5 * Constants.SECOND_MS;
+    long start = System.currentTimeMillis();
+    acquireThread.join(timeoutMs);
+    if (System.currentTimeMillis() - start > timeoutMs) {
```

Comment:
Sure, at `timeoutMs` the result is undefined, but I think `>=` is safer since under normal operation the elapsed time should be very small compared to `timeoutMs`